workflowitems.md

```diff
@@ -122,6 +122,19 @@ In addition, it allows in future to change the 1:1 association between collectio
 
 It returns the workflowitems created by the specified submitter
 
+## Get Single Workflow Item from Item UUID
+
+**/api/submission/workflowitems/item/<:uuid>**
+
+https://dspace7.4science.cloud/server/api/submission/workflowitems/item/cd67ce0e-7f9a-42fc-b8e7-c8bb83ef58ca
+The item uuid is mandatory
+
+There's always at most one workflow item per Item.
+
+It would respond with:
+* The workflow item if there's a match
+* 404 if the workflow item doesn't exist
```

Comment:
Tiny comment here, but I'm assuming it also could respond with a 401 if not logged in, or a 403 if you are logged in but don't have access to the workflow item.  So, I think these should be similar status codes to what you've listed for other endpoints.